src/Metadata/Extractor/XmlResourceExtractor.php

```diff
@@ -156,6 +161,91 @@ private function buildFormats(\SimpleXMLElement $resource, string $key): ?array
         return $data;
     }
 
+    private function buildOpenapi(\SimpleXMLElement $resource): bool|OpenApiOperation|null
+    {
+        if (!isset($resource->openapi) && !isset($resource['openapi'])) {
+            return null;
+        }
+
+        if (\is_bool($resource['openapi'] ?? null) || \in_array((string) $resource['openapi'] ?? null, ['1', '0', 'true', 'false'], true)) {
+            return $this->phpize($resource, 'openapi', 'bool');
+        }
+
+        $openapi = $resource->openapi;
+        $data = [];
+        $attributes = $openapi->attributes();
+        foreach ($attributes as $attribute) {
+            $data[$attribute->getName()] = $this->phpize($attributes, 'deprecated', 'deprecated' === $attribute->getName() ? 'bool' : 'string');
+        }
+
+        $data['tags'] = $this->buildArrayValue($resource, 'tag');
+
+        if (isset($openapi->responses->response)) {
```

Comment:
How does this work?

Reply:
🤷‍♀️ welcome to the wonderful world of XML